tethys_cli/manage_commands.py

```diff
@@ -8,22 +8,28 @@
 ********************************************************************************
 """
 
-from tethys_cli.cli_helpers import get_manage_path, run_process
+from django.core.management import get_commands
+
+from tethys_cli.cli_helpers import get_manage_path, load_apps, run_process
 
 MANAGE_START = 'start'
 MANAGE_COLLECTSTATIC = 'collectstatic'
 MANAGE_COLLECTWORKSPACES = 'collectworkspaces'
 MANAGE_COLLECT = 'collectall'
 MANAGE_CREATESUPERUSER = 'createsuperuser'
-MANAGE_SHELL = 'shell'
+MANAGE_GET_PATH = 'path'
 
 
 def add_manage_parser(subparsers):
-    # Setup start server command
+    # sub-command choices
+    TETHYS_COMMANDS = [MANAGE_START, MANAGE_COLLECTSTATIC, MANAGE_COLLECTWORKSPACES, MANAGE_COLLECT,
+                       MANAGE_CREATESUPERUSER, MANAGE_GET_PATH]
```

Comment:
I realized the `MANAGE_CREATESUPERUSER` is just calling the `manage.py createsuperuser` command, so we should take the option out of `TETHYS_COMMANDS`.

Reply:
done